code/game/objects/items/devices/scanners.dm

```diff
@@ -632,3 +632,253 @@ REAGENT SCANNER
+	dat += "[extra_font]\t-Respiratory Damage %: [target.getOxyLoss()]</font><br>"
+
+	extra_font = (target.getToxLoss() < 60 ? "<font color='blue'>" : "<font color='red'>")
+	dat += "[extra_font]\t-Toxin Content %: [target.getToxLoss()]</font><br>"
+
+	extra_font = (target.getFireLoss() < 60 ? "<font color='blue'>" : "<font color='red'>")
+	dat += "[extra_font]\t-Burn Severity %: [target.getFireLoss()]</font><br>"
+
+	extra_font = (target.radiation < 10 ?"<font color='blue'>" : "<font color='red'>")
+	dat += "[extra_font]\tRadiation Level %: [target.radiation]</font><br>"
+
+	extra_font = (target.getCloneLoss() < 1 ?"<font color='blue'>" : "<font color='red'>")
+	dat += "[extra_font]\tGenetic Tissue Damage %: [target.getCloneLoss()]<br>"
+
+	extra_font = (target.getBrainLoss() < 1 ?"<font color='blue'>" : "<font color='red'>")
+	dat += "[extra_font]\tApprox. Brain Damage %: [target.getBrainLoss()]<br>"
+
+	dat += "Paralysis Summary %: [target.paralysis] ([round(target.paralysis / 4)] seconds left!)<br>"
+	dat += "Body Temperature: [target.bodytemperature-T0C]&deg;C ([target.bodytemperature*1.8-459.67]&deg;F)<br>"
+
+	dat += "<hr>"
+
+	if(target.has_brain_worms())
+		dat += "Large growth detected in frontal lobe, possibly cancerous. Surgical removal is recommended.<br>"
+
+	var/blood_percent =  round((target.blood_volume / BLOOD_VOLUME_NORMAL))
+	blood_percent *= 100
+
+	extra_font = (target.blood_volume > 448 ? "<font color='blue'>" : "<font color='red'>")
+	dat += "[extra_font]\tBlood Level %: [blood_percent] ([target.blood_volume] units)</font><br>"
+
+	if(target.reagents)
+		dat += "Epinephrine units: [target.reagents.get_reagent_amount("Epinephrine")] units<BR>"
+		dat += "Ether: [target.reagents.get_reagent_amount("ether")] units<BR>"
+
+		extra_font = (target.reagents.get_reagent_amount("silver_sulfadiazine") < 30 ? "<font color='black'>" : "<font color='red'>")
+		dat += "[extra_font]\tSilver Sulfadiazine: [target.reagents.get_reagent_amount("silver_sulfadiazine")]</font><br>"
+
+		extra_font = (target.reagents.get_reagent_amount("styptic_powder") < 30 ? "<font color='black'>" : "<font color='red'>")
+		dat += "[extra_font]\tStyptic Powder: [target.reagents.get_reagent_amount("styptic_powder")] units<BR>"
+
+		extra_font = (target.reagents.get_reagent_amount("salbutamol") < 30 ? "<font color='black'>" : "<font color='red'>")
+		dat += "[extra_font]\tSalbutamol: [target.reagents.get_reagent_amount("salbutamol")] units<BR>"
+
+	dat += "<hr><table border='1'>"
+	dat += "<tr>"
+	dat += "<th>Organ</th>"
+	dat += "<th>Burn Damage</th>"
+	dat += "<th>Brute Damage</th>"
+	dat += "<th>Other Wounds</th>"
+	dat += "</tr>"
+
+	for(var/obj/item/organ/external/e in target.bodyparts)
+		dat += "<tr>"
+		var/AN = ""
+		var/open = ""
+		var/infected = ""
+		var/robot = ""
+		var/imp = ""
+		var/bled = ""
+		var/splint = ""
+		var/internal_bleeding = ""
+		var/lung_ruptured = ""

```

Comment:
I've been away from the codebase for a while.... but I remember getting annoyed with Fox for removing lung rupture. Is this back?

Reply:
As above, this is copied from the bodyscanner code. The associated procs to retrieve this value are still there.